data/calculated_parser/functions.py

```
@@ -69,11 +70,467 @@ def sspeed(depth, latitude, temperature, salinity):
+
+    note: sca - Sound Channel Axis
+    """
+    
+    sca_value = np.nanmin(speed)
+    if np.isnan(sca_value):
+        return np.nan
+
+    idx = np.where(speed == sca_value)    
+    idx = int(idx[0][0])
+
+    if idx is 0:
```

Ruff (0.12.2)

Use `==` to compare constant literals

Replace `is` with `==`

(F632)